x-pack/test/functional/apps/painless_lab/painless_lab_flyout.ts

```diff
@@ -5,12 +5,28 @@
  * 2.0.
  */
 
+import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
+const TEST_SCRIPT = `return 1;`;
+const TEST_SCRIPT_REQUEST = `POST _scripts/painless/_execute
+{
+  "script": {
+    "source": """return 1;""",
```

Comment:
```suggestion
    "source": """${TEST_SCRIPT}""",
```

I'd probably interpolate the var in this case, just a nit I guess.